coderd/database/queries/aibridge.sql

```diff
@@ -366,3 +366,25 @@ SELECT (
   (SELECT COUNT(*) FROM user_prompts) +
   (SELECT COUNT(*) FROM interceptions)
 )::bigint as total_deleted;
+
+-- name: ListAIBridgeModels :many
+SELECT
+	model
+FROM
+	aibridge_interceptions
+WHERE
+	ended_at IS NOT NULL
+	-- Filter model
+	AND CASE
+		WHEN @model::text != '' THEN aibridge_interceptions.model ILIKE '%' || @model::text || '%'
```

Comment:
I'm a bit confused why we would want to filter model by model name? There shouldn't be _that_ many unique models? Could we return all models and maybe only filter in frontend?

Problem with filtering by text in backend is efficiency. If I understand correctly, current code will do full table scan.
To use existing B-tree index on `model` column it would need to be case sensitive prefix search: `LIKE @model::text || '%'`

or different index needs to be created: https://www.postgresql.org/docs/current/pgtrgm.html#PGTRGM-INDEX